App/Sagas/TextileSagas.js

```diff
@@ -220,6 +203,7 @@ export function * pairNewDevice (action) {
 
 export function * shareImage ({thread, hash, caption}) {
   try {
+    // TODO: sharePhoto returns a String. Should we be using it?
```

Comment:
the return string here is the newly created block id, we can remove that at some point because it doesn't seem useful since you just request all photos again